SimplyBadCode.cls

```diff
@@ -0,0 +1,8 @@
+class SimplyBadCode {
+  public static void loopExample(){
```

Comment:
How do you see developers using this method? Does it make sense to keep it static? would an instance method make sense coupled with a static wrapper? 

We tend to strive away from static methods, if only because A,B,C.